modules/katamari/src/test/ts/atomic/api/type/TypeTest.ts

```diff
@@ -1,176 +1,177 @@
-  checkIsObject(true, {});
-  checkIsObject(false, objectString);
-  checkIsObject(false, []);
-  checkIsObject(false, noop);
-  checkIsObject(false, [ 1, 3, 4, 5 ]);
-  checkIsObject(false, 1);
-
-  checkIsArray(false, null);
-  checkIsArray(false, undefined);
-  checkIsArray(false, true);
-  checkIsArray(false, false);
-  checkIsArray(false, 'ball');
-  checkIsArray(false, {});
-  checkIsArray(false, objectString);
-  checkIsArray(true, []);
-  checkIsArray(false, noop);
-  checkIsArray(true, [ 1, 3, 4, 5 ]);
-  checkIsArray(false, 1);
-
-  checkIsFunction(false, null);
-  checkIsFunction(false, undefined);
-  checkIsFunction(false, true);
-  checkIsFunction(false, false);
-  checkIsFunction(false, 'ball');
-  checkIsFunction(false, {});
-  checkIsFunction(false, objectString);
-  checkIsFunction(false, []);
-  checkIsFunction(true, noop);
-  checkIsFunction(false, [ 1, 3, 4, 5 ]);
-  checkIsFunction(false, 1);
-
-  checkIsNumber(false, null);
-  checkIsNumber(false, undefined);
-  checkIsNumber(false, true);
-  checkIsNumber(false, false);
-  checkIsNumber(false, 'ball');
-  checkIsNumber(false, {});
-  checkIsNumber(false, objectString);
-  checkIsNumber(false, []);
-  checkIsNumber(false, noop);
-  checkIsNumber(false, [ 1, 3, 4, 5 ]);
-  checkIsNumber(true, 1);
-
-});
-
-UnitTest.test('Type.is*: only one should match for every value', () => {
-  const classifiers = [
-    Type.isString,
-    Type.isObject,
-    Type.isArray,
-    Type.isNull,
-    Type.isBoolean,
-    Type.isUndefined,
-    Type.isFunction,
-    Type.isNumber
-  ];
-
-  fc.assert(fc.property(fc.anything(), (x) => {
-    const matches = Arr.filter(classifiers, (c) => c(x));
-    Assert.eq('number of matching types', 1, matches.length, tNumber);
-  }));
-});
-
-UnitTest.test('Type.isNullable', () => {
-  Assert.eq('null should be nullable', true, Type.isNullable(null));
-  Assert.eq('undefined should be nullable', true, Type.isNullable(undefined));
-  fc.assert(fc.property(fc.string(), (s) => {
-    Assert.eq('string should not be nullable', false, Type.isNullable(s));
-  }));
-  fc.assert(fc.property(fc.integer(), (i) => {
-    Assert.eq('integer should not be nullable', false, Type.isNullable(i));
-  }));
-});
-
-UnitTest.test('Type.isNonNullable', () => {
-  Assert.eq('null should be nullable', false, Type.isNonNullable(null));
-  Assert.eq('undefined should be nullable', false, Type.isNonNullable(undefined));
-  fc.assert(fc.property(fc.string(), (s) => {
-    Assert.eq('string should not be nullable', true, Type.isNonNullable(s));
-  }));
-  fc.assert(fc.property(fc.integer(), (i) => {
-    Assert.eq('integer should not be nullable', true, Type.isNonNullable(i));
-  }));
-
-  // this is testing a compile-time check of the type guard
-  const os: string | null | undefined = 'hello';
-  if (Type.isNonNullable(os)) {
-    const s: string = os;
-    Assert.eq('s', s, os);
-  }
+describe('atomic.katamari.api.struct.TypeTest', () => {
+  it('Type.is* unit tests', () => {
```

Comment:
This is another big `it` block. It'd be good to break this up into an `it` block per `Type.is*` test.

Reply:
Done.